docs/azure_jumpstart_hcibox/RB/_index.md

```diff
@@ -1,65 +1,80 @@
 ---
 type: docs
-linkTitle: "Resource Bridge"
-weight: 3
+linkTitle: "VM Management"
+weight: 7
 ---
 
 ## Virtual machine provisioning with Azure Arc
 
-Azure Stack HCI supports [VM provisioning the Azure portal](https://learn.microsoft.com/azure-stack/hci/manage/azure-arc-enabled-virtual-machines). HCIBox is pre-configured with [Arc resource bridge](https://learn.microsoft.com/azure-stack/hci/manage/azure-arc-enabled-virtual-machines#what-is-azure-arc-resource-bridge) to support this capability.
+Azure Stack HCI supports [VM provisioning the Azure portal](https://learn.microsoft.com/azure-stack/hci/manage/manage-arc-virtual-machines). Like all Azure Stack HCI clusters, the HCIBox cluster comes preconfigured with the components needed for VM management through Azure portal. Follow this guide to configure a basic VM from a marketplace image.
```

Comment:
```suggestion
Azure Stack HCI supports [VM provisioning the Azure portal](https://learn.microsoft.com/azure-stack/hci/manage/manage-arc-virtual-machines). Like all Azure Stack HCI clusters, the HCIBox cluster comes preconfigured with the components needed for VM management through the Azure portal. Follow this guide to configure a basic VM from a marketplace image.
```

Reply:
For some reason i thought we weren't supposed to call it "the Azure portal" just "Azure portal". i could be wrong

Reply:
Azure portal

Reply:
leaving as is